tests/test_ujson.py

```diff
@@ -828,6 +830,39 @@ def test_encode_none_key():
     assert ujson.dumps(data) == '{"null":null}'
 
 
+def test_default_function():
+    iso8601_time_format = "%Y-%m-%dT%H:%M:%S.%f"
+
+    class CustomObject:
+        pass
+
+    class UnjsonableObject:
+        pass
+
+    def default(value):
+        if isinstance(value, datetime.datetime):
+            return value.strftime(iso8601_time_format)
+        elif isinstance(value, uuid.UUID):
+            return value.hex
+        elif isinstance(value, CustomObject):
+            raise ValueError("invalid value")
+        return value
+
+    dt = datetime.datetime.now()
+    assert ujson.dumps(dt, default=default) == '"%s"' % dt.strftime(iso8601_time_format)
```

Comment:
```suggestion
    now = dt.datetime.now()
    assert ujson.dumps(dt, default=default) == '"%s"' % now.strftime(iso8601_time_format)
```

Reply:
@hugovk , I have modified it according to your suggestion.